apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -43,6 +43,40 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'oauth-apps.update',
+	{
+		authRequired: true,
+		validateParams: isUpdateOAuthAppParams,
+	},
```

Comment:
I feel would be good to have the permission check here, so we avoid calling the meteor method if the user actually doesn't have any perm

That would also make the endpoint return a 403 instead of the 400 when validating inside the method :)